### DISCUSSION TITLE
Support Python in GitHub packaging

### DISCUSSION THREAD

**Discussion by InonS** [community/community#37723] [Actions]
[GitHub packages](https://github.com/features/packages) currently support Docker, npm, Maven, NuGet, and RubyGems, but not Python packages, despite the fact that [Python is by far the most popular programming language on GitHub](https://madnight.github.io/githut/#/pull_requests/2022/3). Supporting Python packaging (@PyPA) would clearly benefit the GitHub user community, and improve code quality and maintainability.

Why does GitHub offer packaging support in less popular languages, but not Python?

**Comment by ossanna16** [comment]
Hi there @InonS and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
